plugins/modules/swim_workflow_manager.py

```diff
@@ -343,6 +343,12 @@
 
   - Added the parameter 'dnac_api_task_timeout', 'dnac_task_poll_interval' options in v6.13.2.
 
+  - additional_info |
```

Comment:
We can include this information in device_role itself

    device_role:
      description: |
        Specifies the device role(s) for tagging or untagging the image as golden.

        Permissible values:
        - 'ALL': Applies the golden tag to all devices, regardless of role.
        - 'UNKNOWN': Tags devices without a specified classification.
        - 'ACCESS': Tags devices that connect end-user devices (e.g., access switches).
        - 'BORDER ROUTER': Tags devices linking different network segments or domains.
        - 'DISTRIBUTION': Tags devices aggregating traffic toward the core.
        - 'CORE': Tags backbone devices handling high-volume network traffic.

        Behavior:
        - If 'device_role' is a single string (e.g., `"ACCESS"`), only that role is tagged as golden.
        - If 'device_role' contains multiple roles (e.g., `"ACCESS,CORE"`), all specified roles are tagged as golden.

        To replace an existing golden tag for a specific role:
        - **Unassign** the tag from the current role (e.g., `ACCESS`).
        - **Assign** the tag to the new role (e.g., `CORE`).

        Examples:
        - device_role: "ACCESS" tags only the `ACCESS` role as golden.
        - device_role: "ACCESS,CORE" tags both `ACCESS` and `CORE` roles as golden.
      type: str